website/docs/guides/test-your-typegraph/index.mdx

```diff
@@ -0,0 +1,113 @@
+import TabItem from "@theme/TabItem";
+import SDKTabs from "@site/src/components/SDKTabs";
+
+import PyPmTabs from "@site/src/components/PythonPackageManagerTabs";
+import TsPmTabs from "@site/src/components/TsPackageManagerTabs";
+
+# Test typegraphs
+
+[Typegraph](/docs/reference/typegraph) is the frontend of the _Metatype_ ecosystem where the app is authored and defined. Typegraph is basically a function that includes the definition of the exposed APIs, the embedded runtimes, database table definitions, functions and other building blocks of the application.
```

Comment:
A few comments :
- the more consistent we remain across the documentation (e.g. pick between SDK and frontend) the easiest it is for people to follow
- I would avoid to speak about the metatype ecosystem to keep things simple, let's just call it metatype
- according to https://diataxis.fr/how-to-guides/, guide should be concentrate on the goal and skip explanation or use links to suggest a way to get more info